src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+
+
+    public Ball getRandomBall() {
+
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int randomNumber = new Random().nextInt(100);
+        ColorSupplier fff = new ColorSupplier();
+
+        Ball bigball = new Ball();
+
+        bigball.setColor(fff.toString());
```

Comment:
You are using 'toString()' method to get a string representation of an enum constant. It's better to use 'name()' method. Please, refer to the checklist.